### PR TITLE
refactor: Viewports state

### PR DESCRIPTION
**Pull request by igoroctaviano**
- [x] Clear up circular references.
- [x] Structure and document the code.
- [x] Remove dom of active viewport from state. (Avoid crash of redux dev tool / big state)
- [x] Use commandsManager in connected components that were using dom references from store.
- [x] Improve jsDocs.

### CODE REVIEW

File: platform/viewer/src/connectedComponents/ConnectedCineDialog.js [hunk 1]
```diff
@@ -16,7 +17,8 @@ const { setViewportSpecificData } = OHIF.redux.actions;
 const mapStateToProps = state => {
   // Get activeViewport's `cine` and `stack`
   const { viewportSpecificData, activeViewportIndex } = state.viewports;
-  const { cine, dom } = viewportSpecificData[activeViewportIndex] || {};
+  const { cine } = viewportSpecificData[activeViewportIndex] || {};
+  const dom = commandsManager.runCommand('getActiveViewportEnabledElement');
```
Comment (dannyrb) [inline platform/viewer/src/connectedComponents/ConnectedCineDialog.js]: Ah. Yes. This right here.
Can we duplicate this in the `vtk` code?
(Maybe not until we inject the commandsManager?)

Comment (igoroctaviano) [reply platform/viewer/src/connectedComponents/ConnectedCineDialog.js]: Question... We should wrap commandsModule like init and inject the commandsManager?

Comment (dannyrb) [reply platform/viewer/src/connectedComponents/ConnectedCineDialog.js]: I think so. We can pass it in the same object we pass serviceManager

File: extensions/vtk/src/commandsModule.js [hunk 2]
```diff
@@ -341,7 +342,7 @@ function getVOIFromCornerstoneViewport(displaySet) {
         windowCenter,
       };
     }
-  }
+  } */
```
[the remark this answers is not in the document]
Comment (igoroctaviano) [reply extensions/vtk/src/commandsModule.js]: Done! thx.

File: platform/viewer/src/connectedComponents/ConnectedCineDialog.js (hunk 1 above)
Comment (igoroctaviano) [reply platform/viewer/src/connectedComponents/ConnectedCineDialog.js]: Got it. Done! thx.

**Pull request merged**